### ISSUE TITLE
Generated TF state is not TF 0.14 compliant / fails to import

### ISSUE THREAD

**Issue by ZachGoldberg** [cloudflare/cf-terraforming#214]
Frustrated user here.  

- Downloaded cf-terraform, so far so good
- Got confused by the difference between api key and api token, ok, thats my fault
- Got cf-terraforming to generate resources, awesome!  But... no tfstate and so the plan wants to recreate all my records, bad!
- Found that the table at the bottom of the README says cf-terraform SHOULD be able to generate a statefile for me for records, great!  
- The statefile is NOT TF14 compliant, and "0.13upgrade" doesnt touch state files.  I tried manually find/replacing the state bits in the generated state file, couldnt get Terraform 0.14 to like it.
- OK, so I have to import resources.  One. at. a. time.  There's no way, that I can find, to get a record ID out of cloudflare in the UI... only the API... thankfully the statefile cf-terraform, which isnt usable by terraform itself, has the record Ids already in a JSON file on my system
- Manually wrote a quick script to parse the terraform file and generate `terraform import` commands for each of my ~200 dns records in the zone
- Manually push the cobbled together state to terraform cloud, now things work

So in the end I got everything to work, but it was a madly frustrating process and it seems like such a small amount of effort to automate the importing of resources, or generate a TF-0.14 compliant state, would make it this much easier.  Even just some documentation that describes the terraform workflow thats supposed to work would've been helpful.

Thanks for listening to my rant.

My quick node script to generate a shell script to run imports of dns records, in case its helpful for anyone else

```
const { readFileSync } = require('fs');
const rawdata = readFileSync('cf-terraform-generated-state.tfstate');
const data = JSON.parse(rawdata);
const records  = data.modules[0].resources;
Object.keys(records).forEach(recordName => {
  const record = records[recordName];
  console.log(`terraform import ${recordName} ${record.primary.attributes.zone_id}/${record.primary.id}`);
});
```

**Comment by burner1024** [comment]
@ZachGoldberg I went through all the same...
Did you import actual zones, by chance? No matter what I try, it just fails.

**Comment by jacobbednarz** [comment]
this will be addressed with some upcoming changes in #231
